packages/client/src/pages/Forum/ForumDiscussion/ForumDiscussionBox/index.tsx

```diff
@@ -0,0 +1,53 @@
+import {Link, useParams} from 'react-router-dom';
+import {PATHS} from 'client/src/routers/name';
+import {Button} from 'client/src/components/Button';
+import {ForumDiscussionColumn} from './ForumDiscussionColumn';
+import styles from './ForumDiscussionBox.module.scss';
+
+const dataMock = [
+	{id: 1, title: 'Sky Wars', name: 'Obi Wan Kenobi', date: 'the 22th of December'},
+	{id: 2, title: 'Благодарность разработчикам', name: 'Евгений Малкин', date: 'вчера'},
+	{id: 3, title: 'Война и мир', name: 'Лев Толстой', date: '1867 год'}
+];
+
+export const ForumDiscussionBox = () => {
+	const topicName = useParams().disccussionTitle;
```

Comment:
Лучше бы забрать через деструктуризацию, хотя тоже не совсем безопасно:
```suggestion
	const { disccussionTitle: { topicName }} = useParams();
```